### PR TITLE
Improve the toast notifcations on Windows

### PR DESCRIPTION
**Pull request by alistair23**

### CODE REVIEW

File: src/UWP/UWP.csproj [hunk 1]
```diff
@@ -145,7 +145,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Acr.UserDialogs">
-      <Version>6.3.10</Version>
+      <Version>6.5.1</Version>
```
Comment (kspearrin) [inline src/UWP/UWP.csproj]: We can't only update UWP here. We should update all projects. I'll look into that and report back

Comment (alistair23) [reply src/UWP/UWP.csproj]: Ok, I don't actually need the update for UWP to work, so I'll update this PR dropping this patch.

**Comment by kspearrin** [comment]
We should probably shift this to the IDeviceAction service on each platform. I'd like to eventually get rid of UserDialogs all together so this is probably a good first step.

**Comment by alistair23** [comment]
Ok. I'm not completely sure what you are looking for with that, so I might leave that up to you. I also have no way to test the other devices. Maybe I can revisit this in the future

**Pull request closed without merging**